mapping/sustainability.yaml

```
@@ -1791,27 +1796,25 @@
 
         Publish job creation estimates:
 
-        1. Add a `Metric` object to the `forecasts` array and set its `.title` to 'Jobs created'.
-        2. Add an `observation` object for each job relationship - stage pair and set `.dimensions` accordingly, i.e.
+        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`forecasts`](project-schema.json,,forecasts) array and set its [`.title`](project-schema.json,/definitions/Metric,title) to 'Jobs created'.
```

```suggestion
        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`forecasts`](project-schema.json,,forecasts) array and set its [`.title`](project-schema.json,/definitions/Metric,title) to "Jobs created".
```
set text rather than a code so double quotes